### PR TITLE
Fix wandb.errors.RequireError as reported in #162

### PR DESCRIPTION
**Pull request by ayulockin**
This PR checks for the `wandb` version to ensure `wandb.require("report-editing")` is used with the right version.

For `wandb` version > 0.13.5 `wandb.require` is not required and for `wandb` version < 0.12.21` it is not supported. 

This PR is to fix the original issue raised in #162 by @asmith26.

### CODE REVIEW

**Comment by LouisCastricato** [comment]
Is this function not actually called?

**Comment by ayulockin** [comment]
Didn't get your question @LouisCastricato.

**Comment by LouisCastricato** [comment]
You added a function a function called get wandb version but you did not add a call to it anywhere. Im a bit confused.

**Review by LouisCastricato — APPROVED** [review]

**Review by jon-tow — CHANGES_REQUESTED** [review]
I left some questions regarding possible API changes with the latest `wandb` version.

On another note, would you be able to update the `configs/sweeps` files to reflect recent changes in hyper-parameter naming (see changes [here](https://github.com/jon-tow/trlx/commit/b99c9ce384bcd1e2e7e0cd210c906cc1125fceb0))? If you're too busy, let me know and I'll push to your fork (with your permission).

File: trlx/ray_tune/wandb.py [hunk 1]
```diff
@@ -7,7 +7,28 @@
...
+        wandb.require("report-editing")
+    elif parse_version(wandb.__version__) < parse_version("0.12.21"):
+        logger.info(
+            "Programmatic Reports are not supported in wandb version < 0.12.21, "
+            "please upgrade wandb to 0.12.21 or higher."
+        )
+    else:
+        pass
+
+check_wandb_version()
```
[the remark this answers is not in the document]
Comment (ayulockin) [reply trlx/ray_tune/wandb.py:30]: I am in favour of pinning to `wandb>=0.13.5`.

**Review by jon-tow — APPROVED** [review]

File: examples/ppo_sentiments.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def reward_fn(samples: List[str]) -> List[float]:
     imdb = load_dataset("imdb", split="test")
     val_prompts = [" ".join(review.split()[:4]) for review in imdb["text"]]
 
-    return trlx.train(
+    trlx.train(
```
Comment (LouisCastricato) [inline examples/ppo_sentiments.py:51]: No it should return.

Comment (jon-tow) [reply examples/ppo_sentiments.py:51]: It breaks RayTune:
```
ray.exceptions.RayTaskError(LiveError): ray::ImplicitFunc.train() (pid=1701167, ip=26.0.134.192, repr=main)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/trainable.py", line 367, in train
    raise skipped from exception_cause(skipped)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/function_trainable.py", line 335, in entrypoint
    return self._trainable_func(
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/ray/tune/trainable/function_trainable.py", line 652, in _trainable_func
    output = fn()
  File "/admin/home-guac/tmp/ayu/trlx/examples/ppo_sentiments.py", line 51, in main
    trlx.train(
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trlx.py", line 107, in train
    trainer.learn()
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trainer/accelerate_base_trainer.py", line 385, in learn
    results = self.evaluate()
  File "/admin/home-guac/tmp/ayu/trlx/trlx/trainer/accelerate_base_trainer.py", line 224, in evaluate
    for prompts in tqdm(self.eval_dataloader, desc="Generating samples"):
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/tqdm/rich.py", line 116, in __init__
    self._prog.__enter__()
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/progress.py", line 1169, in __enter__
    self.start()
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/progress.py", line 1160, in start
    self.live.start(refresh=True)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/live.py", line 113, in start
    self.console.set_live(self)
  File "/admin/home-guac/trlx/.env/lib/python3.8/site-packages/rich/console.py", line 840, in set_live
    raise errors.LiveError("Only one live display may be active at once")
rich.errors.LiveError: Only one live display may be active at once
```

and every other example we have does not return the trainer.

Comment (LouisCastricato) [reply examples/ppo_sentiments.py:51]: Got it.

**Comment by jon-tow** [comment]
Thanks @ayulockin !

**Pull request merged**